src/controllers/profile.ts

```diff
@@ -47,15 +95,15 @@ export const getCurrentProfile = async (req: Request, res: Response) => {
 };
 // check up on updating name, dob, and zodiac sign. Is the code underneath correct?
 export const updateProfile = async (req: Request, res: Response) => {
-  const { gender, reasons, location } = req.body;
+  const { gender, reasons, location, zodiacSign } = req.body;
```

Comment:
as far as i know we don't get zodiac sign from the front end, there is a function to calculate it on the backend